sbsp_frontend/src/components/pc/CueList.vue

```
@@ -347,24 +169,30 @@ const onArrowDown = throttle((e: KeyboardEvent) => {
-useHotkey('shift+arrowdown', onArrowDown);
+useHotkey('ArrowDown', onArrowDown);
+useHotkey('Shift+ArrowDown', onArrowDown);
 
-useHotkey('cmd+a', () => {
+useHotkey('$mod+A', () => {
   // This operation not set uiState.selected. But selecting all will includes uiState.selected
   uiState.selectedRows.clear();
-  showModel.flatCueList.filter(item => !item.isHidden).forEach(value => uiState.selectedRows.add(value.cue.id));
+  showModel.flatCueList
+    .filter((item) => !item.isHidden)
+    .forEach((value) => uiState.selectedRows.add(value.cue.id));
```

Biome (2.5.0)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)